charts/backstage/Chart.yaml

```diff
@@ -12,7 +12,11 @@ annotations:
   charts.openshift.io/provider: Red Hat Developer Hub Team
   charts.openshift.io/supportURL: https://github.com/redhat-developer/rhdh-chart/issues
 apiVersion: v2
-description: A Helm chart for deploying RHDH (a Backstage application)
+description: |
```

Comment:
A version bump of the Chart is needed after this change, which is why CI is failing: https://github.com/redhat-developer/rhdh-chart/actions/runs/9307789304/job/25644060264?pr=15

See https://github.com/04kash/rhdh-chart/blob/telemetry-opt-in-banner/charts/backstage/Chart.yaml#L45-L48

Reply:
Done, bumped version to 2.16.0